recipe/meta.yaml

```diff
@@ -1,13 +1,14 @@
 {% set name = "dask-gateway" %}
-{% set version = "0.8.0" %}
+{% set version = "0.9.0" %}
 
 package:
   version: {{ version }}
   name: {{ name|lower }}
 
 source:
   url: https://github.com/dask/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: b7dd60b37953b804bebb999f97263cd0f3493d985f9d922d5e89ee52edf17234
+  sha256: de99927a7f2e97f53b4c0ab7a784fc36ccf5dba22fb1cb15c5bd7d29e5fe1990
+
```

Comment:
Github won't let me comment on lines outside the diff. You'll need to delete the `entry_points` below, these are gone in this release.

Reply:
Oop, there's also some CLI tests below as well that should be deleted.

Reply:
Whoops, I even made a mental note to remove those, but got forgot about it when checking if dask-gateway-server was still a command.